src/quest/manager.py

```diff
@@ -52,7 +59,9 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
             await historian.suspend()
 
     def _get_workflow(self, workflow_id: str):
-        return self._workflows[workflow_id]  # TODO check for key, throw error
+        if (alias := self._check_alias(workflow_id)) is not None:
+            return self._workflows[alias]
+        return self._workflows[workflow_id]
```

Comment:
What do you think of this approach instead?
```py
workflow_id = self._alias_dictionary.get(alias, workflow_id)
return self._workflows[workflow_id]
```
You can follow a similar pattern in the other methods, then delete `_check_workflow_alias`